solutions/devsprint-vinicius-carvalho-1/DeliveryAppChallenge/Screens/Settings/SettingsViewController.swift

```diff
@@ -9,6 +9,8 @@ import UIKit
 
 class SettingsViewController: UIViewController {
 
+    var viewModel: SettingsViewModel = SettingsViewModel()
```

Comment:
Uma outra dica é que você pode passar ou inicializar o seu viewModel por init(), sem precisar instanciar diretamente assim.

Poderia ser algo assim

```
let viewModel: SettingsViewModel

init() {
       super.init(nibName: nil, bundle: nil)
        viewModel.fetch()
        viewModel.view = self
        settingsView.tableViewDelegate = self
}
```

é um exemplo ta, só pra ver como ficaria essa comunicação com o ViewModel no caso aqui, porque em questão nem estamos tratando disso, mas para quando for implementar estar ciente.